test/aeso_compiler_tests.erl

```diff
@@ -221,9 +204,6 @@ compilable_contracts() ->
      "test" % Custom general-purpose test file. Keep it last on the list.
     ].
 
-not_compilable_on(fate) -> [];
-not_compilable_on(aevm) -> compilable_contracts().
```

Comment:
Since there are no non-compatible contracts now, would it make sense to have test all contracts inside `test/contracts` instead of listing all contracts names in the function `compilable_contracts`?

Reply:
No, there are contracts that don't compile for tests that check compiler errors

Reply:
Right, I missed that.